waku/waku_archive/archive.nim

```diff
@@ -92,38 +89,24 @@ proc handleMessage*(
     waku_archive_errors.inc(labelValues = [error])
     return
 
-  let
-    msgDigest = computeDigest(msg)
-    msgDigestHex = msgDigest.data.to0xHex()
-    msgHash = computeMessageHash(pubsubTopic, msg)
-    msgHashHex = msgHash.to0xHex()
-    msgTimestamp =
-      if msg.timestamp > 0:
-        msg.timestamp
-      else:
-        getNanosecondTime(getTime().toUnixFloat())
-
-  trace "handling message",
-    msg_hash = msgHashHex,
-    pubsubTopic = pubsubTopic,
-    contentTopic = msg.contentTopic,
-    msgTimestamp = msg.timestamp,
-    usedTimestamp = msgTimestamp,
-    digest = msgDigestHex
+  let msgHash = computeMessageHash(pubsubTopic, msg)
 
   let insertStartTime = getTime().toUnixFloat()
 
-  (await self.driver.put(pubsubTopic, msg, msgDigest, msgHash, msgTimestamp)).isOkOr:
+  (await self.driver.put(msgHash, pubsubTopic, msg)).isOkOr:
     waku_archive_errors.inc(labelValues = [insertFailure])
-    error "failed to insert message", error = error
-
-  debug "message archived",
-    msg_hash = msgHashHex,
+    trace "failed to insert message",
+      hash = msgHash.to0xHex(),
+      pubsubTopic = pubsubTopic,
+      contentTopic = msg.contentTopic,
+      timestamp = msg.timestamp,
+      error = error
+
+  trace "message archived",
+    hash = msgHash.to0xHex(),
```

Comment:
We should always keep the `msg_hash` attribute so that we can always grep easily :)
```suggestion
    msg_hash = msgHash.to0xHex(),
```